lib/good_job/cron_entry.rb

```diff
@@ -96,13 +99,23 @@ def display_properties
       {
         key: key,
         class: job_class,
-        cron: cron,
+        cron: schedule,
         set: display_property(set),
         args: display_property(args),
         description: display_property(description),
       }
     end
 
+    def fugit
+      @fugit ||= begin
+        parsed_cron = Fugit.parse(cron)
+        
+        raise ArgumentError, "Invalid cron format: '#{cron}'" unless parsed_cron.instance_of?(Fugit::Cron)
```

Comment:
I think this ArgumentError would be clearer if placed in #initialize.

Reply:
I agree :) Updated. And fixed linting errors.